Input.css

```diff
@@ -0,0 +1,94 @@
+@import"Css/main.css";
+.high{
+    color:red
+}
+.active{
+    width:40vw;
+    height:fit-content;
+    background-color: white;
+    border-radius: 5px;
+    padding:2rem;
+    margin:3rem;
+}
+form{
```

Comment:
Styling an element directly is not a good idea. As the user might not want the same style for every element.

Reply:
okay mahesh from next time will keep in mind